src/applications/vaos/tests/e2e/workflows/appointment-list-workflow/past-appointments.cypress.spec.js

```diff
@@ -98,7 +98,7 @@ describe('VAOS past appointment flow', () => {
       // Act
       PastAppointmentListPageObject.visit()
         .assertAppointmentList({ numberOfAppointments: 1 })
-        .selectDateRange(2);
+        .selectDateRange('2');
```

Comment:
Updating to uswds V3, generates `-select` as index 0 in the dropdown. Changing to use the date range value instead of its index